client/js/app.modules/main/header.html

```diff
@@ -0,0 +1,11 @@
+<header class="header page__header" ng-class="{color_blue: isProjectView}">
+    <a href="" ng-click="isNavHide = !isNavHide" class="header__menu-button button">
```

Comment:
Move isNavHide assignment to controller. This kind of logic should be done there controller.